lib/index.d.ts

```diff
@@ -27,6 +27,14 @@ export type SessionOptions = {
 
 export type Handler = (req: any, res: any) => any;
 
+export type Session = {
+  set: <T = any>(name: string, value: T) => void;
```

Comment:
should we add that:
- set sends back the data you set
?

or is this already handled?

Reply:
@vvo you mean jsdoc comments? no, they are not there. I could probably copy them from the source package, but I didn't see any comment here https://github.com/vvo/next-iron-session/blob/master/lib/index.js#L58-L75 so I thought they are not required here